store/CSDK/test/main_ft.cpp

```diff
@@ -0,0 +1,1172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
```

Comment:
Can you split different function test to different files?
maybe can refer: https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core/framework, https://github.com/tensorflow/tensorflow/blob/81fa63d5dd34b9a8046f79c16731efa4dee9dd63/tensorflow/core/BUILD

Reply:
It is good idea to arrange test case in different files .but since it is a framework code. Can we add this Testcase refactoring in separate PR (i will add JIRA for this) ?

Reply:
ok